components/home/menu/calendar/index.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+// import apiCalendar from 'react-google-calendar-api';
+import {
+  View, Text, Button, TouchableOpacity
+} from 'react-native';
+import { createAppContainer, createSwitchNavigator } from 'react-navigation';
+import firebase from '../../../../firebase.config';
+import LoginScreen from './Screens/LoginScreen';
+import LoadingScreen from './Screens/LoadingScreen';
+import DashboardScreen from './Screens/DashboardScreen';
+
+
+export default class Calendar extends Component {
+  constructor(props){
+    super(props);
+  }
+
+// g'2020-03-24T00:00:00+00:00'
```

Comment:
what does this refer to

Reply:
We were testing different date formats because Google Calendar API requires using RFC3339 timestamp for writing dates. The following is the correct format: YYYY-MM-DDTHH:MM:SS.MMMZ. For example: 2020-01-07T01:01:01.000Z

Reply:
okay good, can you write a comment to describe that this is the format.
ie: 
// use this format for method parameter <FORMAT>

Reply:
Done!